scripts/zones/Selbina/DefaultActions.lua

```diff
@@ -3,6 +3,7 @@ local ID = require("scripts/zones/Selbina/IDs")
 return {
     ['Abelard']      = { text = ID.text.ABELARD_DIALOG },
     ['Elfriede']     = { event = 25 },
+    ['Devean']       = { event = ID.npc.DEVEAN_DEFAULT },
```

Comment:
And this will need to be updated to ID.text.DEVEAN_DEFAULT

Reply:
Changed to use number.